### ISSUE TITLE
ruby-beautify ignores Rubocop "Exclude" flags

### ISSUE THREAD

**Issue by bbugh** [Glavin001/atom-beautify#1773]
# Description

Given that I have this in my `.rubocop.yml` file:

```yaml
Style/BlockDelimiters:
  Exclude:
    - "spec/**/*"
```

I expect that atom-beautify would not run the Style/BlockDelimiters cop on my spec files. 

Unfortunately that's not the case. When running a beautification, it ignores the Exclude setting, even though running `rubocop` from the command line works properly.

# Input Before Beautification

This is what the code looked like before:

```ruby
it "raises an error" do
  expect {
    get something_path
  }.to raise_error ArgumentError
end
```

# Expected Output

It should not change from the input.

# Actual Output

The beautified code actually looked like this:

```ruby
it "raises an error" do
  expect do
    get something_path
  end.to raise_error ArgumentError
end
```

This meant that Style/BlockDelimiters ran on the spec. This does not happen when running rubocop from the command line.

# Steps to Reproduce

1. Add this to the project `.rubocop.yml`
    ```ruby
    Style/BlockDelimiters:
      Exclude:
        - "spec/**/*"
    ```

2. Create a file like this under `/spec`:
    ```ruby
    # /spec/example_spec.rb
    describe "something" do
      it "raises an error" do 
        expect {
          # whatever
        }.to raise_error ArgumentError
      end
    end
    ```

2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code should not change!

# Debug

The Debug file outputs a lot of NDA-violating content, so I can't post it.

# Checklist

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [ ] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by bbugh** [comment]
I looked at the code, and this is happening because of how the temp file is written - it does not preserve the path and does not set the working directory, so Rubocop does not exclude the files as expected when it is run. Here it is simply writing a temp file:

```coffee
          # src/beautifiers/rubocop.coffee:54
          "--config", configFile
          tempFile = @tempFile("temp", text, '.rb')
          ], {ignoreReturnCode: true})
```

However, this should be writing out to the relative path of the file with the correct file name, so that excludes are not ignored. Something like `path.join(tempDir, relativePath)`. Running the command also needs to run on the working directory of the temp directory.

**Comment by Glavin001** [comment]
Thanks for reporting this issue and diving into the code to investigate further! Pull Requests welcome are always and it would be great to improve Rubocop support. Thanks in advance!

**Comment by Glavin001** [comment]
Published to v0.32.0
